src/yarpmanager/src-manager/mainwindow.cpp

```diff
@@ -70,6 +71,8 @@
 
 using namespace std;
 
+constexpr int32_t win10_1909{18363}; // see https://en.wikipedia.org/wiki/Windows_10#Updates_and_support
```

Comment:
Can we use a permanent URL? Either the Windows docs, or a link to a permanent wiki version (see https://en.wikipedia.org/wiki/Help:URL#WP)? Otherwise a Wikipedia article is always bound to change.

Reply:
There is a handy new [URL shortener extension](https://meta.wikimedia.org/wiki/Wikimedia_URL_Shortener) for MediaWiki: https://meta.wikimedia.org/wiki/Special:UrlShortener + https://w.wiki/E62.